sdcm/coredump.py

```diff
@@ -381,7 +382,10 @@ def update_coredump_info_with_more_information(self, core_info: CoreDumpInfo):
                             # On some systems two digit timezone is not recognized as correct timezone
                             time_spat[3] = f'{timezone}00'
                             timestring = ' '.join(time_spat)
-                        fmt = "%a %Y-%m-%d %H:%M:%S %z"
+                        if re.search(r'^[A-Za-z]*$', timezone):
```

Comment:
if you think it might be only a Loader/Centos specific case - it may be usefull to add a comment about it

Reply:
it's not, it's more depended on the configured time on a machine, and probably the version of coredumpctl, either way we need to support it regardless of which machine/distribution it is.